root/etc/cont-init.d/98-nvm

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/with-contenv bash
+
+echo "**** installing nvm ****"
+export NVM_DIR="$HOME/.nvm"
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
+
+echo "**** loading nvm ****"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
The install will automatically adds the load to `.bashrc` and `.zshrc`. 

Every new instance of `bash` or `zsh` should start with nvm (if everything went well...)

We are only loading the nvm at this point to be able to run the script bellow and install the first node version, which I think is cool.

But is also an option remove it (after line 6) and then start without any node version

Reply:
can you change this into a proper if block?

Reply:
Yes I can, like this?

```suggestion
if [ -s "$NVM_DIR/nvm.sh" ]; then
  \. "$NVM_DIR/nvm.sh"
fi
```

To be honest I dont know if is necessary the if, like we are installing right above
Can we just load it?

```suggestion
\. "$NVM_DIR/nvm.sh"
```

Reply:
I guess. One question, does the install script need to run on every start? Is that how it is supposed to be updated?

Reply:
No, does not need to run every start. Just once.

What is needed to load every start is the `nvm.sh` but the install script adds it to .bashrc and .zshrc

Reply:
Then you should put that install operation inside the if block so it only installs it the first time. NVM folder is in the config folder and should be persistent

Reply:
Changed